guides/whats-new.md

```diff
@@ -0,0 +1,214 @@
+# What's New
+
+We're always adding and updating the Corvid APIs, and we want you to have one place to find all of our latest changes and cool additions.
+
+## 🗓️ October 13, 2020
+### Updated: Added information that wixLocation.to() does not work in preview
```

Comment:
For Updated and Fix items, some of the subheadings state "Added information" and some of them do not. I would suggest leaving it out of the subheading and just stating the fact (wixLocation.to() does not work in preview) and then in the description below to use "Added a note" or "Added information".